_posts/2020-06-05-cruise-control.md

```diff
@@ -0,0 +1,213 @@
+```
+
+This will show an output like that shown below:
+
+```yaml
+Name:         my-rebalance
+Namespace:    myproject
+Labels:       strimzi.io/cluster=my-cluster
+Annotations:  API Version:  kafka.strimzi.io/v1alpha1
+Kind:         KafkaRebalance
+Metadata:
+# ...
+Status:
+  Conditions:
+    Last Transition Time:  2020-06-04T14:36:11.900Z
+    Status:                ProposalReady
+    Type:                  State
+  Observed Generation:     1
+  Optimization Result:
+    Data To Move MB:  0
+    Excluded Brokers For Leadership:
+    Excluded Brokers For Replica Move:
+    Excluded Topics:
+    Intra Broker Data To Move MB:         12
+    Monitored Partitions Percentage:      100
+    Num Intra Broker Replica Movements:   0
+    Num Leader Movements:                 24
+    Num Replica Movements:                55
+    On Demand Balancedness Score After:   82.91290759174306
+    On Demand Balancedness Score Before:  78.01176356230222
+    Recent Windows:                       5
+  Session Id:                             a4f833bd-2055-4213-bfdd-ad21f95bf184
+```
+
+The key parts to pay attention to here are the `Status.Conditions.Status` field, which shows the progress of your rebalance request (if the proposal is not ready yet you may see `ProposalPending` as the `Status`) and the `Status.Optimization Result` which (once the proposal is ready) will show a summary of the optimization proposal. The meaning of each of these values is described in the [Strimzi documentation](https://strimzi.io/docs/operators/latest/using.html#con-optimization-proposals-str), essentially this summary shows you the scope of the changes that Cruise Control has suggested and from this you can figure out what kind of impact these changes may have on your cluster whilst they are being applied.
+
+#### Interpreting an optimization proposal
+
+In the case shown above, Cruise Control has suggested 55 partition replicas (totalling 12MB of data) should be move between separate brokers (_inter_-broker) in the cluster. This will involve moving data across the network, but 12MB is not a lot so this should not have too much of an impact. If this number was very high you may want to time this rebalance for a period of less load, as moving large amounts of data could effect the performance of the cluster as a whole. 
+
+There are no _intra_ (between disks on the same broker) movements proposed. These kind of movements only involve movement between disks so have no network impact, but may effect the performance the particular brokers whose disks are exchanging data. 
+
+Finally, this proposal is suggesting 24 leader replica changes. This is the cheapest change that can be applied as it only involves a configuration change in Zookeeper.
+
+### Starting a cluster rebalance
+
+If you are happy with the proposal then you can apply the `approve` annotation to the `KafkaRebalance` resource to tell Cruise Control to start applying the changes:
```

Comment:
I would say "the approve value to the strimzi.io/rebalance annotation". Wdyt? Or something like that.